wgpu-hal/src/dx12/device.rs

```diff
@@ -897,6 +904,9 @@ impl crate::Device for super::Device {
             bind_cbv.space += 1;
         }
 
+        let mut dynamic_storage_buffer_offsets_targets = std::collections::BTreeMap::new();
+        let mut total_dynamic_storage_buffers = 0;
```

Comment:
God this whole thing is going to be one massive brutal conflict with #6766 😅

Reply:
Oops sorry this is yours to deal with :(

Reply:
It's fine - you opened your PR first.